charts/prometheus-operator/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+description: The chart deployes the Promethes Operator
+icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png
+engine: gotpl
+type: application
+maintainers:
+  - name: langecode
+    email: tal@netic.dk
+  - name: bismarck
+name: prometheus-operator
+sources:
+  - https://github.com/prometheus-community/helm-charts
+  - https://github.com/prometheus-operator/prometheus-operator
+version: 0.2.0
```

Comment:
why are you starting with 0.2.0?

Reply:
Good catch - I think it was just a leftover - should probably just start out `1.0.0`